frontend/src/old-pages/Configure/Configure.tsx

```diff
@@ -302,78 +125,145 @@ function Configure() {
 
   useWizardSectionChangeLog()
 
+  const handleSubmit = useCallback(() => {
+    wizardHandleCreate(() => clearWizardState(clearState, false), navigate)
+  }, [wizardHandleCreate, clearWizardState, navigate])
+
+  const handleRefresh = () => {
+    setRefreshing(true)
+    let region = getState(['wizard', 'region'])
+    let chosenRegion = region === 'Default' ? null : region
+    LoadAwsConfig(chosenRegion, () => setRefreshing(false))
+  }
+
+  const navigateWizard = useWizardNavigation(validate)
+
+  const handleOnNavigate: NonCancelableEventHandler<WizardProps.NavigateDetail> =
+    useCallback(
+      ({detail}) => {
+        navigateWizard(detail.reason, detail.requestedStepIndex)
+      },
+      [navigateWizard],
+    )
+
+  const showSecondaryActions = useCallback(() => {
```

Comment:
nit-pick: no need to adopt `useCallback` here, since you are interested in the returned value

I think we can get away without memoizing this component for now

Reply:
Ok, will remove the memoization for now